RockLib.Messaging.SQS/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 	
 #### Added
 - Added `.editorconfig` and `Directory.Build.props` files to ensure consistency.
+- Added "SQS.MessageID" as header to SQSReceiverMessages when the not attempting to unpack as an SNS message
```

Comment:
A couple issues at the end of the line with `unpack as an SNS message`.  Can you update it to `unpack as a SQS message.`.

Reply:
Thats actually correct the code is supports to content of the sqs message as an SNS message I guess.  https://github.com/RockLib/RockLib.Messaging/blob/main/RockLib.Messaging.SQS/SQSReceiverMessage.cs#L51

Reply:
I noticed I had a mistake in my correction, and you caught it. Nice!